### PR TITLE
add aliases for add/remove icons

### PR DESCRIPTION
**Pull request by VasylHryha**
## Description
close https://github.com/epicmaxco/vuestic-ui/issues/4058

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Comment by VasylHryha** [comment]
Should we retain the `decreaseIcon` and `increaseIcon` props, or should we completely transition to using `iconConfig` for defining icons in `VaCounter` ?

**Comment by m0ksem** [comment]
> Should we retain the `decreaseIcon` and `increaseIcon` props, or should we completely transition to using `iconConfig` for defining icons in `VaCounter` ?

We don't do this for other component icons like Select arrow down, Input error icon, Modal close icon. 
We need to make rules how to use icons in a component and then standardize this behavior across all components.

File: packages/ui/src/components/va-counter/VaCounter.vue [hunk 1]
```diff
@@ -101,7 +101,7 @@ import { toRefs, computed, shallowRef, InputHTMLAttributes, ComputedRef, toRef,
 import omit from 'lodash/omit'
 import pick from 'lodash/pick'
 
-import { safeCSSLength } from '../../utils/css'
+import { safeCSSLength } from '@/utils/css'
```
Comment (m0ksem) [inline packages/ui/src/components/va-counter/VaCounter.vue:104]:
```suggestion
import { safeCSSLength } from '../../utils/css'
```

We use relative paths only.

**Review by m0ksem — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/va-counter/VaCounter.vue (hunk 1 above)
Comment (VasylHryha) [reply packages/ui/src/components/va-counter/VaCounter.vue]: reverted changes

**Review by m0ksem — APPROVED** [review]

**Pull request merged**